src/content/en/fundamentals/engage-and-retain/push-notifications/handling-messages.markdown

```diff
@@ -178,7 +181,7 @@ self.addEventListener('push', function(event) {
 });
 {% endhighlight %}
 
-Check for notifications that match `data.tag` with a call to `getNotifications()`.
+Once we have the message data, call `getNotifications()` using `data.tag`.
```

Comment:
I'm going to take this as a separate PR. I need to figure out how tag is added to a message and put it in sending-messages.md.